Gemfile

```diff
@@ -74,6 +75,7 @@ end
 
```

Comment:
- [ ] It would be nice if you ask your partner to commit some code in your code because I saw that your partner didn't push any code from his side it would be great if @Benawi push some major code in your pull request to make your pull request fulfill the pair-programming requirement 👍